addons/payment/views/ir_module_module_views.xml

```diff
@@ -0,0 +1,85 @@
+                                    <img type="open" t-att-src="kanban_image('ir.module.module', 'icon_image', record.icon_image.value)"/>
+                                </t>
+                            </div>
+                            <div class="o_payment_acquirer_desc">
+                                <h3 class="mt4"><t t-esc="record.shortdesc.value"/></h3>
+                                <t t-if="record.description_html.raw_value" t-raw="record.description_html.raw_value"/>
+                            </div>
+                            <div class="o_payment_acquirer_bottom">
+                                <t t-if="record.state.raw_value == 'installed'">
+                                    <field name="payment_environment" widget="kanban_label_selection" options="{'classes': {'test': 'warning', 'production': 'success'}}"/>
+                                </t>
+                                <button t-if="record.state.raw_value != 'installed'" type="object" class="btn btn-default pull-right btn-sm" name="action_payment_immediate_install">Install</button>
+                                <button t-if="record.state.raw_value == 'installed'" type="object" class="btn btn-primary pull-right btn-sm" name="action_payment_edit">Configure</button>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="ir_module_module_view_form_payment" model="ir.ui.view">
+        <field name="name">ir.module.module.view.form.payment</field>
+        <field name="model">ir.module.module</field>
+        <field name="arch" type="xml">
+            <form string="Payment Modules">
+            </form>
+        </field>
+    </record>
+
+    <record id="ir_module_module_view_tree_payment" model="ir.ui.view">
+        <field name="name">ir.module.module.view.tree.payment</field>
+        <field name="model">ir.module.module</field>
+        <field name="arch" type="xml">
+            <tree string="Payment Modules">
+                <field name="state"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="ir_module_module_action_payment" model="ir.actions.act_window">
+        <field name="name">Payment Acquirers</field>
+        <field name="domain">[('name', 'ilike', 'payment_')]</field>
+        <field name="res_model">ir.module.module</field>
+        <field name='view_type'>form</field>
+        <field name='view_mode'>kanban,tree,form</field>
+        <field name="view_ids"
+               eval="[(5, 0, 0),
+                      (0, 0, {'view_mode': 'kanban', 'view_id': ref('ir_module_module_view_kanban_payment')}),
+                      (0, 0, {'view_mode': 'tree', 'view_id': ref('ir_module_module_view_tree_payment')}),
+                      (0, 0, {'view_mode': 'form', 'view_id': ref('ir_module_module_view_form_payment')})
+                     ]"/>
```

Comment:
🚫 view_ids is a super bad ideas, I've made that mistake, it's a PITA when needing to alter or upgrade them. Better to create explicit `ir.actions.act_window.view` via `<record>` tags.